config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  root 'questions#index'
```

Comment:
`root` goes at the bottom because the routes file is hierarchical from top to bottom.